<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calcul d'alcoolémie</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="text">
       <string>Calcul d'alcoolémie</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="genderLayout">
      <item>
       <widget class="QLabel" name="genderLabel">
        <property name="text">
         <string>Sexe</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="maleRadio">
        <property name="text">
         <string>Homme</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="femaleRadio">
        <property name="text">
         <string>Femme</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="weightLayout">
      <item>
       <widget class="QLabel" name="weightLabel">
        <property name="text">
         <string>Poids</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="weightInput"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>6</number>
      </property>
      <item row="5" column="0">
       <widget class="QLabel" name="aperoLabel">
        <property name="text">
         <string>Apéritif</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="beerLabel">
        <property name="text">
         <string>Bière</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="beerVolumeLabel">
        <property name="text">
         <string>5°</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="AperoQuantityInput"/>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelVolume">
        <property name="text">
         <string>% en volume</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelQuantity">
        <property name="text">
         <string>Quantité en cl</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="AperoVolumeLabel">
        <property name="text">
         <string>35°</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="vineLabel">
        <property name="text">
         <string>Vin</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelBoissons">
        <property name="text">
         <string>Boissons</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="vineVolumeLabel">
        <property name="text">
         <string>12°</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="VineQuantityInput"/>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="beerQuantityInput"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>Effacer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="validateButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Valider</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="resultLabel"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
